geth/account/keyfile_finder.go

```diff
@@ -0,0 +1,51 @@
+package account
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ethereum/go-ethereum/common"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+var keyfileFinder keyFileFinder = keyFileFinderBase{}
+
+type keyFileFinderBase struct{}
+
+func (kf keyFileFinderBase) Find(keyStoreDir string, addressObj common.Address) ([]byte, error) {
```

Comment:
When introducing new exported types/methods, let's make it `godoc` friendly.

Reply:
Absolutely.

Reply:
Do we need comments for unexported types? I think nope.

Reply:
Unfortunate development since generation of web documentation for public identifiers with JavaDoc or today GoDoc (and others). Sure that those special comments help to get a good and quick overview within a browser. 

But the comments for private identifiers address others. They are for the maintainers trying to understand what the original coder thought. So here it would be nice to read what `Find()` does, what it expects as arguments, how it reacts in positive cases, and how in negative ones.

I know, I'm pretty pedantic here, sorry. But I went through huge code bases during my career and often lost time navigating through the code to get aware, what the author could have written in two or three sentences. And it also shows what a caller of the function expects and should not be changed if not needed due to the maintenance issue.

Reply:
I've added comments.